server/services/permissionsService.test.ts

```diff
@@ -0,0 +1,785 @@
+        },
+      )
+
+      it.each([Role.SocDataAnalyst, Role.SocDataManager])(
+        'should return view false, edit true if user has %s role',
+        async role => {
+          const user = { ...prisonUserMock, userRoles: [role] }
+
+          const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+          expect(permissions.soc).toEqual({
+            view: false,
+            edit: true,
+          })
+        },
+      )
+
+      it('should return view and edit false if user has no pathfinder roles', async () => {
+        const permissions = await service.getOverviewPermissions(prisonUserMock, PrisonerMockDataA)
+
+        expect(permissions.soc).toEqual({
+          view: false,
+          edit: false,
+        })
+      })
+    })
+
+    describe('offenderCategorisation', () => {
+      it.each([
+        Role.CreateCategorisation,
+        Role.ApproveCategorisation,
+        Role.CreateRecategorisation,
+        Role.CategorisationSecurity,
+      ])('should return edit true if user has %s role', async role => {
+        const user = { ...prisonUserMock, userRoles: [role] }
+
+        const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+        expect(permissions.offenderCategorisation).toEqual({
+          view: false,
+          edit: true,
+        })
+      })
+
+      it('should return view and edit false if user has no offenderCategorisation roles', async () => {
+        const permissions = await service.getOverviewPermissions(prisonUserMock, PrisonerMockDataA)
+
+        expect(permissions.offenderCategorisation).toEqual({
+          view: false,
+          edit: false,
+        })
+      })
+    })
+
+    describe('probationDocuments', () => {
+      it.each([Role.PomUser, Role.ViewProbationDocuments])(
+        'should return view true if user has %s role and prisoner in caseload',
+        async role => {
+          const user = { ...prisonUserMock, userRoles: [role] }
+
+          const permissions = await service.getOverviewPermissions(user, PrisonerMockDataA)
+
+          expect(permissions.probationDocuments).toEqual({
+            view: true,
+          })
+        },
+      )
+
+      it('should return view false if user has no probationDocuments roles', async () => {
+        const permissions = await service.getOverviewPermissions(prisonUserMock, PrisonerMockDataA)
+
+        expect(permissions.probationDocuments).toEqual({
+          view: false,
+        })
+      })
+
+      it('should return view false if user not in caseload', async () => {
+        const user = { ...prisonUserMock, userRoles: [Role.GlobalSearch, Role.ViewProbationDocuments] }
+        const prisoner = { ...PrisonerMockDataA, prisonId: 'LEI' }
+
+        const permissions = await service.getOverviewPermissions(user, prisoner)
+
+        expect(permissions.probationDocuments).toEqual({
+          view: false,
+        })
+      })
+    })
```

Comment:
Think we're missing a test case `should return view true if user has %s role and prisoner is %s`

with OUT and TRN as the final options. since it looks like if they've got the right role and the prisoner is out or transferring they can still view